rackcity/models/it_model.py

```diff
@@ -1,26 +1,16 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from rackcity.models.fields import RCPositiveIntegerField
-import re
 from django.contrib.postgres.fields import ArrayField
-
-
-DEFAULT_DISPLAY_COLOR = "#394B59"
-
-
-def validate_display_color(value):
-    if value:
-        color_pattern = re.compile("#[A-Fa-f0-9]{6}")
-        if color_pattern.fullmatch(value) is None:
-            raise ValidationError(value + " is not a valid hex color")
+from rackcity.models.model_utils import DEFAULT_DISPLAY_COLOR,validate_display_color
```

Comment:
`DEFAULT_DISPLAY_COLOR,validate_display_color` did you lint 🧐

Reply:
BEN LINTER STRIKES AGAIN